osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -580,12 +582,24 @@ private void load(OsuColour colours, AudioManager audioManager, INotificationOve
 
                     volumeOverlay.IsMuted.Value = false;
 
+                    double aggregateVolumeTrack = audioManager.Volume.Value * audioManager.VolumeTrack.Value;
+
                     // Check values before resetting, as the user may have only had mute enabled, in which case we might not need to adjust volumes.
                     // Note that we only restore halfway to ensure the user isn't suddenly overloaded by unexpectedly high volume.
-                    if (audioManager.Volume.Value <= volume_requirement)
-                        audioManager.Volume.Value = 0.5f;
-                    if (audioManager.VolumeTrack.Value <= volume_requirement)
-                        audioManager.VolumeTrack.Value = 0.5f;
+                    if (aggregateVolumeTrack <= volume_requirement)
```

Comment:
This is all so black box to a user. I'd still prefer the old hard cut increases, then let the user adjust further.

Reply:
Sounds fine, I agree it's a touch more complicated/arbitrary but wasn't sure if it warrants a discussion.